plugins/inventory/proxmox.py

```diff
@@ -92,7 +92,17 @@
         default: proxmox_
         type: str
       want_facts:
-        description: Gather LXC/QEMU configuration facts.
+        description:
+          - Gather LXC/QEMU configuration facts.
+          - When I(want_facts) is set to C(true) more details about QEMU VM status are possible, besides the running and stopped states.
+            Currently if the VM is running and it is suspended, the status will be running and the machine will be in C(running) group,
+            but it's actual state will be paused. This introduces multiple groups [prefixed with I(group_prefix)] C(prelaunch) and C(paused).
```

Comment:
```suggestion
            but its actual state will be paused. This introduces multiple groups [prefixed with I(group_prefix)] C(prelaunch) and C(paused).
```
But shouldn't this behavior change description should be in the `qemu_extended_statuses` description?